mesonbuild/interpreter/compiler.py

```diff
@@ -1,4 +1,5 @@
 import functools
+from mesonbuild.interpreterbase.decorators import typed_kwargs, KwargInfo
```

Comment:
we usually use relative imports, so `from ..interpreterbase.decoartors`

Reply:
Yeah my language server snuck that in via auto-import, will adjust.